services/brig/src/Brig/User/Auth.hs

```diff
@@ -110,7 +110,7 @@ lookupLoginCode phone =
       Data.lookupLoginCode u
 
 login :: Login -> CookieType -> ExceptT LoginError AppIO (Access ZAuth.User)
-login (PasswordLogin li pw label) typ = do
+login (PasswordLogin li pw label _) typ = do
```

Comment:
You could implement something like a hole here for the case that the feature flag is set:

```suggestion
login (PasswordLogin li pw label _) typ = do
  case TeamFeatureSndFPasswordChallengeNotImplemented of
    -- mark this place to implement handling verification codes later
    -- (for now just ignore them unconditionally.)
    _ -> pure ()
```

And introduce that type in `/libs/wire-api/src/Wire/API/Team/Feature.hs` where we will later implement the feature config.  Once we do, the compiler will show us all the places that we need to touch.

```haskell
data TeamFeatureSndFPasswordChallengeNotImplemented
  = TeamFeatureSndFPasswordChallengeNotImplemented
```

But I may be overthinking things?